hcs/swagger.json

```diff
@@ -1419,13 +1419,9 @@
           "$ref": "#/definitions/hashicorp.cloud.consulama.ama.Boolean",
           "description": "enabled should be set to TRUE to enable audit logging."
         },
-        "managedIdentity": {
```

Comment:
Why is this unnecessary? Did something else change in AMA to retrieve the identity during the create workflow?